src/views/common/Animation/AnimationControlButtonStyles.css

```diff
@@ -1,6 +1,12 @@
 div.AnimationControlButton span {
-    padding-left: 4px;
-    padding-right: 4px;
+    padding: 4px;
+    /* text-align: center; /* Ought to work, but doesn't, alas. */
```

Comment:
For some reason this styling isn't honored properly when I put it in the CSS instead of in explicit properties.